$OpenBSD$

OpenBSD patch for patch-splitmix-cbits-unix-init_c.

$OpenBSD$

Remove the unused Linux-specific sys/random.h include.
OpenBSD provides getentropy(2) via unistd.h.

--- splitmix-0.1.3.1/cbits-unix/init.c.orig
+++ splitmix-0.1.3.1/cbits-unix/init.c
@@ -1,4 +1,3 @@
 #include <stdint.h>
 #include <unistd.h>
-#include <sys/random.h>
 
 uint64_t splitmix_init() {
 	uint64_t result;
